src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -481,6 +503,11 @@ void manualRegistrationToolBox::disableSaveButtons(bool param)
     d->b_exportTransformation->setDisabled(param);
 }
 
+void manualRegistrationToolBox::disableComputeButton(bool param)
+{
+    d->b_computeRegistration->setDisabled(param);
+}
```

Comment:
(and doing that we are more robust in case the language change. If "setDisabled" becomes "setNewDisabled" we can adapt our code easily)